src/test/java/core/basesyntax/db/StorageTest.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.db;
+
+import static org.junit.jupiter.api.Assertions.assertInstanceOf;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertSame;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+class StorageTest {
+
+    @AfterEach
+    void tearDown() {
+        Storage.getFruitStorage().clear();
+    }
+
+    @Test
+    void getFruitStorage_getInstance_Ok() {
+        Map<String, Integer> storage = Storage.getFruitStorage();
+        assertNotNull(storage);
+        assertSame(storage, Storage.getFruitStorage());
+    }
+
+    @Test
+    void getFruitStorage_isHashMap_Ok() {
+        assertInstanceOf(HashMap.class, Storage.getFruitStorage(),
+                "Expected instance of HashMap");
+    }
+
+    @Test
+    void storage_acceptsStringAndInteger_Ok() {
```

Comment:
Similar case here - you're checking something trivial. 
If your storage implements some specific add/delete operation, it would be reasonable to compare the state before & after the operation, but we're dealing with a regular hash map here.